applications/system/js_app/js_modules.c

```diff
@@ -267,6 +267,8 @@ void js_check_sdk_compatibility(struct mjs* mjs) {
 
 static const char* extra_features[] = {
     "baseline", // dummy "feature"
+    "gpio-pwm",
+    "widget-view",
```

Comment:
```suggestion
    "gui-widget",
```
Just so it's based on module name?

I'm just nitpicking at this point, amazing work overall 💯